tools/python/boutiques/schema/example_good.json

```diff
@@ -0,0 +1,236 @@
+  "schema-version": "0.4",
+  "walltime-estimate": 3600,
+  "groups": [
+    {
+      "id": "parcellation_group",
+      "name": "Group of all used parcellations",
+      "members": [
+        "desikan",
+        "jhu",
+        "talairach",
+        "aal",
+        "harvardoxford",
+        "cpac200"
+      ],
+      "one-is-required": true
+    }
+  ],
+  "inputs": [
+    {
+      "id": "dti_file",
+      "name": "Diffusion Tensor Image",
+      "type": "File",
+      "value-key": "[DTI]",
+      "optional": false
+    },
+    {
+      "id": "bval_file",
+      "name": "B-values file",
+      "type": "File",
+      "value-key": "[BVAL]",
+      "optional": false
+    },
+    {
+      "id": "bvec_file",
+      "name": "Gradient vectors file",
+      "type": "File",
+      "value-key": "[BVEC]",
+      "optional": false
+    },
+    {
+      "id": "mprage_file",
+      "name": "Structural scan file",
+      "type": "File",
+      "value-key": "[MPRAGE]",
+      "optional": false
+    },
+    {
+      "id": "atlas",
+      "name": "Atlas image (MNI152)",
+      "type": "String",
+      "value-key": "[ATLAS]",
+      "optional": false,
+      "value-choices": [
+        "/ndmg_atlases/atlas/MNI152_T1_1mm.nii.gz"
+      ]
+    },
+    {
+      "id": "mask",
+      "name": "Atlas brain mask",
+      "type": "String",
+      "value-key": "[MASK]",
+      "optional": false,
+      "value-choices": [
+        "/ndmg_atlases/atlas/MNI152_T1_1mm_brain_mask.nii.gz"
+      ]
+    },
+    {
+      "id": "outdir",
+      "name": "Output directory",
+      "type": "String",
+      "value-key": "[OUTDIR]",
+      "optional": false
+    },
+    {
+      "id": "desikan",
+      "name": "Desikan parcellation",
+      "type": "String",
+      "value-key": "[DESIKAN]",
+      "optional": true,
+      "value-choices": [
+        "/ndmg_atlases/labels/desikan.nii.gz"
+      ]
+    },
+    {
+      "id": "jhu",
+      "name": "JHU parcellation",
+      "type": "String",
+      "value-key": "[JHU]",
+      "optional": true,
+      "value-choices": [
+        "/ndmg_atlases/labels/JHU.nii.gz"
+      ]
+    },
+    {
+      "id": "talairach",
+      "name": "Talairach parcellation",
+      "type": "String",
+      "value-key": "[TALAIRACH]",
+      "optional": true,
+      "value-choices": [
+        "/ndmg_atlases/labels/Talairach.nii.gz"
+      ]
+    },
+    {
+      "id": "aal",
+      "name": "AAL parcellation",
+      "type": "String",
+      "value-key": "[AAL]",
+      "optional": true,
+      "value-choices": [
+        "/ndmg_atlases/labels/AAL.nii.gz"
+      ]
+    },
+    {
+      "id": "harvardoxford",
+      "name": "Harvard-Oxford parcellation",
+      "type": "String",
+      "value-key": "[HARVARDOXFORD]",
+      "optional": true,
+      "value-choices": [
+        "/ndmg_atlases/labels/HarvardOxford.nii.gz"
+      ]
+    },
+    {
+      "id": "cpac200",
+      "name": "CPAC200 parcellation",
+      "type": "String",
+      "value-key": "[CPAC200]",
```

Comment:
Do you want the schema to be backwards compatible and support this key, or not? If so, I added it as well to the schema and can push that, but if you want it to be caught as an error, without any changes it was :)